exporter/prometheusexporter/accumulator.go

```diff
@@ -327,3 +334,81 @@ func copyMetricMetadata(metric pmetric.Metric) pmetric.Metric {
 
 	return m
 }
+
+func AccumulateNumericValues(prev, current, dest pmetric.NumberDataPoint) {
+	if current.StartTimestamp().AsTime().Before(prev.StartTimestamp().AsTime()) {
+		dest.SetStartTimestamp(current.StartTimestamp())
+	} else {
+		dest.SetStartTimestamp(prev.StartTimestamp())
+	}
+
+	older := prev
+	newer := current
+	if current.Timestamp().AsTime().Before(prev.Timestamp().AsTime()) {
+		older = current
+		newer = prev
+	}
+
+	newer.Attributes().CopyTo(dest.Attributes())
+	dest.SetTimestamp(newer.Timestamp())
+
+	switch newer.ValueType() {
+	case pmetric.NumberDataPointValueTypeDouble:
+		dest.SetDoubleValue(newer.DoubleValue() + older.DoubleValue())
+	case pmetric.NumberDataPointValueTypeInt:
+		dest.SetIntValue(newer.IntValue() + older.IntValue())
+	}
+}
+
+func AccumulateHistogramValues(prev, current, dest pmetric.HistogramDataPoint) {
```

Comment:
This should not be exported.

```suggestion
func accumulateHistogramValues(prev, current, dest pmetric.HistogramDataPoint) {
```

Reply:
Updated.